src/bio/bio_internal.h

```diff
@@ -60,15 +60,15 @@ struct bio_dma_buffer {
 enum bio_bs_state {
 	/* Healthy and fully functional */
 	BIO_BS_STATE_NORMAL	= 0,
-	/* Being marked as faulty by monitor, rebuilding pool targets */
+	/* Being detected as faulty */
 	BIO_BS_STATE_FAULTY,
-	/* Rebuild triggered, tearing down SPDK resources */
+	/* Affected targets are marked as DOWN, safe to tear down blobstore */
```

Comment:
keep rebuild triggered?

Reply:
'mark targets as DOWN' is more accurate than 'rebuild triggered'. We just need to make sure that target is DOWN and no IO will come through, if the rebuild process is triggered or not by the rebuild module doesn't matter.